Description: Raise elasticsearch version to 1.0.2.RC1
 Raise elasticsearch version to 1.0.2.RC1
Author: Andreas Wirooks <andreas.wirooks@1und1.de>

--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 #   wget or curl
 #
 JRUBY_VERSION=1.7.8
-ELASTICSEARCH_VERSION=0.90.9
+ELASTICSEARCH_VERSION=1.0.0.RC1
 
 WITH_JRUBY=java -jar $(shell pwd)/$(JRUBY) -S
 JRUBY=vendor/jar/jruby-complete-$(JRUBY_VERSION).jar
